src/modules/inspect/process-list/process-list.ts

```diff
@@ -30,7 +30,7 @@ export class ProcessList {
   private _activeSolutionUri: string;
   private _router: Router;
 
-  private _getCorrelationsIntervalId: number;
+  private timeout: NodeJS.Timer;
```

Comment:
You could increase readability by improving the name of this variable

```suggestion
  private _pollingTimeout: NodeJS.Timer;
```